### ISSUE TITLE
[REVIEW]: ELDAM: A Python software for Life Cycle Inventory data management

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2765]
**Submitting author:** @GustaveCoste (<a href="http://orcid.org/0000-0001-5867-0507">Gustave Coste</a>)
**Repository:** <a href="https://framagit.org/GustaveCoste/eldam" target ="_blank">https://framagit.org/GustaveCoste/eldam</a>
**Version:** v1.0
**Editor:** @sjpfenninger
**Reviewer:** @konstantinstadler, @romainsacchi
**Archive:** <a href="https://doi.org/10.15454/6EKXJQ" target="_blank">10.15454/6EKXJQ</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/905491355d2bda381255aef7c7d39d05/status.svg)](https://joss.theoj.org/papers/905491355d2bda381255aef7c7d39d05)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/905491355d2bda381255aef7c7d39d05"><img src="https://joss.theoj.org/papers/905491355d2bda381255aef7c7d39d05/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/905491355d2bda381255aef7c7d39d05/status.svg)](https://joss.theoj.org/papers/905491355d2bda381255aef7c7d39d05)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://framagit.org/GustaveCoste/eldam" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@konstantinstadler & @romainsacchi, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @sjpfenninger know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://framagit.org/GustaveCoste/eldam">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@GustaveCoste) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://framagit.org/GustaveCoste/eldam">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@GustaveCoste) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @konstantinstadler, @romainsacchi it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
PDF failed to compile for issue #2765 with the following error: 

 Can't find any papers to compile :-(

**Comment by sjpfenninger** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: :point_left:

**Comment by romainsacchi** [comment]
It is a very interesting tool, and will certainly be useful to the LCA community. Well done!
I have followed the instructions, tested it with a couple of dummy files and it produced the expected results.

I have nevertheless the three following issue (issues 1 and 2 being somewhat related).
While they are not of extreme importance, they would greatly improve the user experience.

Issue 1 - https://framagit.org/GustaveCoste/eldam/-/issues/291
Issue 2 - https://framagit.org/GustaveCoste/eldam/-/issues/292
Issue 3 - https://framagit.org/GustaveCoste/eldam/-/issues/293

**Comment by romainsacchi** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: :point_left:

**Comment by romainsacchi** [comment]
The author has adequately answered all the issues I raised.

**Comment by konstantinstadler** [comment]
@whedon and @sjpfenninger : I can not mark the check-marks although I am logged in and accepted the invitation - can you help me or should I just describe my answers?

**Comment by whedon** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

```
@whedon commands
```

**Comment by konstantinstadler** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List Whedon's capabilities
@whedon commands

# List of editor GitHub usernames
@whedon list editors

# List of reviewers together with programming language preferences and domain expertise
@whedon list reviewers

EDITORIAL TASKS

# Compile the paper
@whedon generate pdf

# Compile the paper from alternative branch
@whedon generate pdf from branch custom-branch-name

# Ask Whedon to check the references for missing DOIs
@whedon check references

# Ask Whedon to check repository statistics for the submitted software
@whedon check repository

```

**Comment by konstantinstadler** [comment]
This is a very interesting tool and I believe it can be useful for LCA practioner. The envisaged workflow tightly coupled with Simapro has the potential to lift the quality of LCI data.

I am, however, not a LCA expert and do not have access to Simapro. As far as possible I tested the software and it works for me but I could not verify the integration with Simapro - I hope that this was covered by @romainsacchi  . Thus, I focused my review on the documentation and code. I have some issues which I think need to be addressed and I took the opportunity to also add some recommendation for the way forward.

Issues:

- https://framagit.org/GustaveCoste/eldam/-/issues/302
- https://framagit.org/GustaveCoste/eldam/-/issues/294
- https://framagit.org/GustaveCoste/eldam/-/issues/297
- https://framagit.org/GustaveCoste/eldam/-/issues/299
- https://framagit.org/GustaveCoste/eldam/-/issues/300
- https://framagit.org/GustaveCoste/eldam/-/issues/301

Recommendations:

- https://framagit.org/GustaveCoste/eldam/-/issues/295
- https://framagit.org/GustaveCoste/eldam/-/issues/296
- https://framagit.org/GustaveCoste/eldam/-/issues/298


One other small point I am not sure how to handle (@sjpfenninger ): The repository lists one developer (in the commit history and as stated in the readme). The article, however, has multiple authors. Probably not a problem but perhaps demands a statement about roles (see also https://joss.readthedocs.io/en/latest/submitting.html#authorship )

**Comment by sjpfenninger** [comment]
@romainsacchi, @konstantinstadler Thanks for your reviews!

**Comment by sjpfenninger** [comment]
@romainsacchi Can you confirm that you have tested the integration with Simapro?

**Comment by sjpfenninger** [comment]
@konstantinstadler Can you check if you can now edit the checkboxes? And it looks like @GustaveCoste addressed your issues, can you confirm whether you are satisfied?

**Comment by sjpfenninger** [comment]
@GustaveCoste Can you clarify the role of the authors on the paper?

**Comment by romainsacchi** [comment]
@sjpfenninger @konstantinstadler no, I cannot say I have tested the integration with Simapro, as I do not have a Simapro license. I only tested the consistency between the inputs and the outputs (from Simapro-compatible CSV file to Excel and vice-versa).

**Comment by GustaveCoste** [comment]
@sjpfenninger ELDAM's code was written entirely by myself but the three other authors have helped to design substantial parts of the tool.

I did not address all @konstantinstadler's issues yet, I should have time to do it next week.

**Comment by konstantinstadler** [comment]
Hi,
1) @GustaveCoste - timeline is ok for me
2) Simapro (@sjpfenninger ): I can probably find someone in the institute using Simapro. I never used it before but perhaps that makes me a perfect guinea pig for following a tutorial. Probably need some time to catch a researcher here not working from home, but when I receive a tutorial from @GustaveCoste I should be able to do that within a week
3) I still can not click the buttons. Do I miss something here - I am just supposed to click, right? I tested on several browsers/os and it does not work. But I can make my own list and it works (https://github.com/konstantinstadler/pymrio_article/issues/2 ) - wonder what is going on. Can we do it manually and I just confirm that I have checked this. Or perhaps repost the list and I try again

**Comment by sjpfenninger** [comment]
@whedon re-invite @konstantinstadler as reviewer

**Comment by whedon** [comment]
OK, the reviewer has been re-invited.

@konstantinstadler please accept the invite by clicking this link: https://github.com/openjournals/joss-reviews/invitations

**Comment by sjpfenninger** [comment]
@konstantinstadler Can you go to the invitation link above -- that should hopefully fix the issue. And, if you can get a colleague to look at the Simapro integration, that would be fantastic (I myself also do not have it). We could add them as a third reviewer here for just that small aspect.

**Comment by sjpfenninger** [comment]
Thanks @GustaveCoste, let us know here when you're through; and thanks for clarifying author contributions, that is fine.

**Comment by sjpfenninger** [comment]
@konstantinstadler Can you confirm whether you can edit the review checklist now and whether you have any progress with your Simapro colleague?

**Comment by konstantinstadler** [comment]
Hi, based on the resolved issues and the new documentation it looks really good - I very much like the new documentation on readthedocs @GustaveCoste 👏🏽 . 
Regarding Simapro: I am at a conference right now until Friday. I will check if I can do that in the next days or early next week latest.

**Comment by sjpfenninger** [comment]
@konstantinstadler Great, I'll check back next week and in the worst case we proceed without testing the Simapro integration.

**Comment by sjpfenninger** [comment]
@whedon remind @sjpfenninger in 1 week

**Comment by whedon** [comment]
Reminder set for @sjpfenninger in 1 week

**Comment by konstantinstadler** [comment]
Ok, I did a round of Simapro-ELDAM-Simapro (based on the Simapro intro LCA: coffeemachine and the ELDAM GUI) - works well. Also, all other points have been adequately answered. So I recommend to publish it. :ok: :+1:

**Comment by sjpfenninger** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
PDF failed to compile for issue #2765 with the following error: 

 Can't find any papers to compile :-(

**Comment by sjpfenninger** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: :point_left:

**Comment by GustaveCoste** [comment]
Thank you @romainsacchi and @konstantinstadler for your very useful remarks. Eldam is probably more professional grade now.

**Comment by sjpfenninger** [comment]
@GustaveCoste I'm emailing you suggested edits to the paper (it sounds like it might take a while for account activation on framagit). One question regarding CI: I see there is a CI configuration in the repository, but do you actually run any CI? And if so where are the results? Could be nice to have a badge that shows that in the README.

**Comment by GustaveCoste** [comment]
@sjpfenninger Good idea, I just added the badge.

**Comment by GustaveCoste** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02765/joss.02765/10.21105.joss.02765.pdf) :page_facing_up: :point_left:

**Comment by sjpfenninger** [comment]
@GustaveCoste One extra thing - I noticed that you distribute two DLL files with the package, what is the reason for that?

**Comment by GustaveCoste** [comment]
It is a workaround of a "compilation" issue of `cx_freeze` (which makes a standalone executable of the python package).

**Comment by sjpfenninger** [comment]
@GustaveCoste Ok, I guess that's unavoidable then. We're almost ready to accept this. Can you make a tagged release on GitHub and archive that release, for example on Zenodo or figshare, then report the version number and archive DOI back here in the review issue?

**Comment by GustaveCoste** [comment]
I planned to make an archive on [data.inrae.fr](https://data.inrae.fr/), the archive service of my institution. Would-it be enough? This is the DOI it will have once published: [10.15454/UPDPDT](https://doi.org/10.15454/UPDPDT).

And is it OK if I make a release on the project repository (hosted on framagit.org) instead of Github? The only drawback I see is that I don't think I could create an automatic link between the archive and the release as with Zenodo/Github, but I can manually add the release link in the archive metadata.

**Comment by sjpfenninger** [comment]
Archive: that is fine. Can you just ensure the title and authors on the archive deposition match the paper title and authors.

Release: yes, of course on the project repository which in your case is not hosted on GitHub.

**Comment by GustaveCoste** [comment]
Is there a way to already have the link to the published paper? I would like to add it in the archive and the release metadata.
[Here is the release on Eldam's repo](https://framagit.org/GustaveCoste/eldam/-/releases/JOSS) and [here is an unpublished draft version of the archive](https://data.inrae.fr/privateurl.xhtml?token=1600e0ac-1ba8-478b-aaa4-87ad30643eb6 ).

**Comment by sjpfenninger** [comment]
I do not believe there is. Can you update the archive with the DOI after publication?

**Comment by GustaveCoste** [comment]
I can but an update of the metadata will generate a new minor version of the archive. It may be better to publish the archive only once the paper url is available.

**Comment by GustaveCoste** [comment]
I just thought that I could add the DOI of the article in the archive instead of an url. @sjpfenninger could you confirm that the DOI of the article is [10.21105/joss.02765](https://doi.org/10.21105/joss.02765)? In that case, I could publish the archive.

**Comment by sjpfenninger** [comment]
Yes, you are correct, the DOI is based on the review issue number. So that would work.

**Comment by sjpfenninger** [comment]
And just to confirm, v1.0 is the right version to be associated with the paper, right?

**Comment by sjpfenninger** [comment]
@whedon set 10.15454/UPDPDT as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.15454/UPDPDT" target="_blank">10.15454/UPDPDT</a> is the archive.

**Comment by GustaveCoste** [comment]
Yes, the version to associate to the paper is the v1.0.
The archive is published.

**Comment by sjpfenninger** [comment]
@whedon set v1.0 as version

**Comment by sjpfenninger** [comment]
@GustaveCoste https://doi.org/10.15454/UPDPDT just takes me to a login page?

**Comment by sjpfenninger** [comment]
I just need to confirm that the author list and title on the archive matches that of the paper then we're ready.

**Comment by GustaveCoste** [comment]
@sjpfenninger apparently there is a bug with the dataset publication. I contacted the support and they will investigate. I will let you know when it is fixed.

**Comment by GustaveCoste** [comment]
@sjpfenninger the publishing bug has been fixed. Normally you can know access the archive.

**Comment by GustaveCoste** [comment]
They just noticed me that they couldn't publish the dataset without changing the DOI, The new DOI is [10.15454/6EKXJQ](http://doi.org/10.15454/6EKXJQ).

**Comment by GustaveCoste** [comment]
@whedon set 10.15454/6EKXJQ as archive

**Comment by whedon** [comment]
I'm sorry @GustaveCoste, I'm afraid I can't do that. That's something only editors are allowed to do.

**Comment by sjpfenninger** [comment]
@GustaveCoste Apologies for the holiday-induced delay. The author list of the archive deposition needs to match the author list of the paper. Can you fix that?

**Comment by GustaveCoste** [comment]
@sjpfenninger I just changed the authors of the archive.

**Comment by sjpfenninger** [comment]
@whedon set 10.15454/6EKXJQ as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.15454/6EKXJQ" target="_blank">10.15454/6EKXJQ</a> is the archive.

**Comment by sjpfenninger** [comment]
Ok, one final issue: the archive says version 1.1, do I need to update the version for the paper to 1.1 too?

**Comment by GustaveCoste** [comment]
The 1.1 version of the archive corresponds to the change in the author list in the metadata (for consistency with the paper). The dataset did not change. I don't think it is necessary to change the version for the paper.

**Comment by sjpfenninger** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #2765 with the following error: 

 Can't find any papers to compile :-(

**Comment by arfon** [comment]
@whedon accept from branch joss

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- None

MISSING DOIs

- None

INVALID DOIs

- https://doi.org/10.1016/j.jclepro.2020.124099 is INVALID because of 'https://doi.org/' prefix
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2027

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2027, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true from branch joss 
```

**Comment by arfon** [comment]
@sjpfenninger - just checking that you're ready to hand off this paper for final checks by the EiC team?

@GustaveCoste - please change the DOI string for the DOI that Whedon has flagged to `10.1016/j.jclepro.2020.124099`

**Comment by sjpfenninger** [comment]
@arfon yes should be good to go!

**Comment by arfon** [comment]
@whedon check references

**Comment by arfon** [comment]
@GustaveCoste - just to confirm, please change [this line](https://framagit.org/GustaveCoste/eldam/-/blob/joss/paper.bib#L26) to: `doi = "10.1016/j.jclepro.2020.124099",`

**Comment by GustaveCoste** [comment]
@arfon Sorry for the delay, I just changed the line.

**Comment by arfon** [comment]
@whedon accept from branch joss

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by arfon** [comment]
@whedon accept from branch joss

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2032

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2032, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true from branch joss 
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2033

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2033, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true from branch joss 
```

**Comment by arfon** [comment]
@whedon accept deposit=true from branch joss

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1351479902915223553) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/2034
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.02765](https://doi.org/10.21105/joss.02765)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by arfon** [comment]
@konstantinstadler, @romainsacchi - many thanks for your reviews here and to @sjpfenninger for editing this submission. JOSS relies upon the volunteer efforts of folks like yourselves, and we simply wouldn't be able to do it without you! ✨ 

@GustaveCoste - your paper is now accepted and published in JOSS :zap::rocket::boom:

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.02765/status.svg)](https://doi.org/10.21105/joss.02765)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.02765">
  <img src="https://joss.theoj.org/papers/10.21105/joss.02765/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.02765/status.svg
   :target: https://doi.org/10.21105/joss.02765
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.02765/status.svg)](https://doi.org/10.21105/joss.02765)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss
